mx.jruby/mx_jruby.py

```diff
@@ -203,6 +107,10 @@ def extractArguments(cli_args):
                 rubyArgs.append(arg)
                 rubyArgs.extend(args)
                 break
+    if classic:
+        main_class = "org.jruby.Main"
+    else:
+        main_class = "org.jruby.truffle.Main"
```

Comment:
classic will not launch anymore if the jar is not on the classpath, so probably this launcher should not support classic anymore.